combirepo/rpm_patcher.py

```diff
@@ -182,13 +182,15 @@ def create_patched_packages(queue):
         return
 
     logging.debug("Chrooting to {0}".format(root))
-    dev_path = os.path.join(root, "dev/")
+    dev_path = os.path.join(root, "dev/null")
```

Comment:
Do you bind single `/dev/null`?
Then rename variable please.

Reply:
Done